exes/mqtt-broadcaster/inc/mqtt_broadcaster.hpp

```diff
@@ -524,7 +631,7 @@ class mqtt_broadcaster {
     logger_.info("Node rebirth metric added to the payload");
 
     for (signal_data& signal_data : signals_) {
-      logger_.info("Processing signal_data: {}", signal_data.information.name);
+      logger_.info("signals: Processing signal_data: {}", signal_data.information.name);
```

Comment:
what catagories would they be seperated into?

Reply:
it would be categorized in the cockpit frontend, it would show as different logger instance

Reply:
I don't fully understand. Do you want many loggers in the class which each have their own keys? From where  would they derive their keys?

Reply:
what I mean is if you would like to use different logger for slots and signals you could make 2 instances for each purpose with different key, instead of prefixing log message with its purpose

Reply:
I don't think it would make logging clearer. There are multiple functions which are shared between both and then it would be unclear which should be used.